src/indexer.ts

```diff
@@ -66,7 +66,7 @@ export class Indexer {
     //console.debug('indexBlock', blockID); // DEBUG
     this.logger.info({ block: { id: blockID } }, `indexing block #${blockID}`);
     const contractID = AccountID.parse(this.config.engine).unwrap();
-
+    let timestamp = Math.floor(Date.now() / 1000); // init timestamp
```

Comment:
This makes indexing nondeterministic: two parallel indexers will not produce the same database entries. That must be fixed. It's better for skip blocks to have no timestamps (it's a nullable column) than to have arbitrary timestamps.